apps/api-gateway/src/public-data/public-data.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Inject, Injectable } from '@nestjs/common';
+import { ClientProxy } from '@nestjs/microservices';
+import { PostCategoryMessagePattern } from '@shared/constants/message-pattern';
+
+@Injectable()
+export class PublicDataService {
+  public constructor(
+    @Inject('PUBLIC_DATA_SERVICE') private readonly publicDataClient: ClientProxy,
+  ) {}
+
+  public getPostData() {
+    const pattern = PostCategoryMessagePattern.GET_POST_DATA;
+    return this.publicDataClient.send(pattern, {});
+  }
+
+  public getOnePostData(id: number) {
```

Comment:
보통 특정 조건에 맞게 1개의 데이터를 가져오는 경우에는 아래와 같이 네이밍을 해주면 더 좋을 것 같습니다.
```suggestion
  public getPostBy(id: number) {
```

```suggestion
  public getPostById(id: number) {
```

Reply:
보다보니가 제가 접미사를 습관적으로 Data를 붙이네요...
변수명이 길어지는게 크게 상관 없다면,
`getPostCategoryById`, `getPostCategories` 같은 식의 이름은 어떻게 생각하시나요?

Reply:
[5b35177](https://github.com/GrebbMe/grabbme-be/pull/13/commits/5b35177b1ab9350b8e465fdcc1246ec969681e7a)

해당 커밋 이후로는 변경 된 method 명을 사용하도록 반영 하였습니다~!

Reply:
제안주신 이름 좋다고 생각합니다~